chain/chain/src/flat_storage_resharder.rs

```diff
@@ -828,18 +925,60 @@ fn copy_kv_to_left_child(
 pub enum FlatStorageReshardingEventStatus {
     /// Split a shard.
     /// Includes the parent shard uid and the operation' status.
-    SplitShard(ShardUId, SplittingParentStatus),
+    SplitShard(ShardUId, SplittingParentStatus, ExecutionStatus),
```

Comment:
Can you clarify what's the difference or rename the different status fields? Is the first one more or a Prepare/ Preprocess/ Schedule to the latter's Execution?